server/room-battle.js

```diff
@@ -997,6 +997,14 @@ class RoomBattle extends RoomGames.RoomGame {
 		return player;
 	}
 
+	forcedPublic() {
```

Comment:
`isForcedPublic` reads better, I think? The codebase is kinda inconsistent about this, though.

Reply:
I agree, but `is` to me implies a `boolean` should be returned, not `string | undefined`.

Reply:
Fair, it can be left as-is.

But note that we use `is` as a prefix for a lot of other pseudo-booleans which can be treated as booleans but also contain other information.

Reply:
I'm fine to change to `isForcedPublic` if you'd prefer, it just seems like it was add to confusion to me. Its a trivial change, if you'd like me to `sed -i s/forcedPublic/isForcedPublic/g` im happy to do so

Reply:
I don't have a strong preference here. I'd say leave it alone for now.